src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -968,6 +969,17 @@ Parameter Plugin::GetMetric(const std::string& name, const std::map<std::string,
     } else if (name == ov::caching_properties) {
         std::vector<ov::PropertyName> cachingProperties;
         return decltype(ov::caching_properties)::value_type(cachingProperties);
+    } else if (name == ov::device::architecture) {
```

Comment:
Several points on blob compatibility checks:
1. We need to check driver version + uarch version + EU config to ensure blob compatibility.
2. device::architecture shall include plugin_name + uarch_str when uarch version is know, and plugin_name + device_name otherwise.
3. All of properties above must be added to the list of sensitive properties for the cache (ov::caching_properties)
   - We already have properties for uarch version and EU count, so just need to add those to caching_properties list
   - for driver version we need an internal property. Please add new header file in the plugin which declares GPU specific properties and will be used inside the plugin. This driver version property will be returned in caching_properties list and will be queried by ov::Core via string, but won't be visible to OV users

Reply:
I updated it as you reviewed. But I don't quite understand what you mean by creating a new header file. If the current implementation is not the direction you want, please provide me a little more detailed explanation. Thank you.

Reply:
We need a new *internal* property for driver version. So you need to add new include file somewhere (here probably src/plugins/intel_gpu/include/intel_gpu/plugin) and declare there a new property like here https://github.com/openvinotoolkit/openvino/blob/master/src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp#L122 and then add this new property to caching_properties list.

Reply:
Can be added in separate PR too I think. Also, please consider using different separator for plugin name and device_name/uarch_str. GPU.12.5.0 looks similar to device IDs